packages/core/src/filters/FilterSystem.js

```diff
@@ -326,26 +332,35 @@ export class FilterSystem extends System
         this.statePool.push(state);
     }
 
+    /**
+     * Binds a renderTexture with corresponding `filterFrame`, clears it if mode corresponds.
+     * @param {PIXI.RenderTexture} filterTexture renderTexture to bind, should belong to filter pool or filter stack
+     * @param {PIXI.CLEAR_MODES} [clearMode] clearMode, by default its CLEAR/YES. See {@link PIXI.CLEAR_MODES}
+     */
+    bindAndClear(filterTexture, clearMode = CLEAR_MODES.CLEAR)
+    {
+        this.renderer.renderTexture.bind(filterTexture, filterTexture ? filterTexture.filterFrame : null);
+        if (clearMode === CLEAR_MODES.CLEAR
+            || (clearMode === CLEAR_MODES.BLIT && this.forceClear)
+            || clearMode === true)
```

Comment:
This is an incorrect/incomplete way to implement backward compatibility. The API shouldn't support both booleans and CLEAR_MODES. Please move this condition out of the if statement so that we can easily remove it later. For instance:

```js
// TODO: remove in next major version
if (typeof clearMode === 'boolean') {
  clearMode = clearMode ? CLEAR_MODES.CLEAR : CLEAR_MODES.BLEND;
  // get deprecation function from utils
  deprecation('5.2.1', 'Use CLEAR_MODES when using clear applyFilter option');
}

```